internal/metamorph/server.go

```diff
@@ -244,6 +244,16 @@ func (s *Server) processTransaction(ctx context.Context, waitForStatus metamorph
 		Txid:   txID,
 		Status: metamorph_api.Status_RECEIVED,
 	}
+
+	for _, cb := range data.Callbacks {
+		if cb.CallbackURL != "" {
+			returnedStatus.Callbacks = append(returnedStatus.Callbacks, &metamorph_api.Callback{
+				CallbackUrl:   cb.CallbackURL,
+				CallbackToken: cb.CallbackToken,
+			})
+		}
+	}
```

Comment:
Is this just for completeness? Because I think that will have no effect

Reply:
For post request, I just include the callback data in the response. The data that was provided in the request.